src/amo/components/Search/styles.scss

```diff
@@ -0,0 +1,56 @@
+@import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
+
+.Search {
+  @include respond-to(large) {
+    display: grid;
+    grid-auto-flow: column dense;
+    grid-gap: 10px;
+    grid-template-columns: minmax(300px, 35%) auto;
+    padding: $padding-page;
+  }
+
+  .Card {
```

Comment:
It adds padding to the cards on the search page, but it might be possible to do without the nesting, I'll see.